kratos/processes/process.h

```diff
@@ -193,6 +193,18 @@ class Process : public Flags
     ///@}
     ///@name Input and output
     ///@{
+    
+    /**
+     * @brief This method returns the specifications of current proces, which might depend on the input settings
+     * @param rParameters Input parameters
+     * @return Specifications of current process
+     */
+    static const Parameters GetSpecifications(const Parameters rParameters)
```

Comment:
why `static` instead of `virtual`?

Reply:
We discussed a lot in the @KratosMultiphysics/technical-committee about this. Long story short, we need to have this accessible before construction so making it virtual would require to have prototypes of the processes (as we do for elements/conditions), something that we want to avoid.

Furthermore, note that the `static` is not that bad in the case of processes as the level of inheritance is rarely larger than one and each process is expected to have its own requirements.